Vasily_Kalugin/lesson03/translater.js

```diff
@@ -0,0 +1,52 @@
+const request = require('request');
+const readline = require('readline');
+
+const rl = readline.createInterface({
+    input: process.stdin,
+    output: process.stdout,
+});
+
+function getTranslation(text) {
+    const promiss = new Promise((pRes, pRej) => {
+        const key = 'trnsl.1.1.20190914T160514Z.bb8d3e64ec3b31be.eb50adcef77d561239c08db909d782c574cdfbca';
+        const lang = 'en-ru';
+        const format = 'plain';
+        const options = '1';
+        const mainPath = 'https://translate.yandex.net/api/v1.5/tr.json/translate';
+        const URL = `${mainPath}?key=${key}&lang=${lang}&text=${encodeURI(text)}&format=${format}&options=${options}`;
+
+        request({ timeout: 3000, uri: URL }, (err, res, body) => {
```

Comment:
Тогда и парсить ничего не нужно будет :)
```suggestion
        request({ timeout: 3000, uri: URL, json: true }, (err, res, body) => {
```

Reply:
+